docs/releases.rst

```diff
@@ -4,6 +4,12 @@
     Releases
 ======================
 
+tmt-1.35
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``tmt try`` now supports :ref:`/stories/cli/try/option/epel` option backed by :ref:`prepare/feature<plugins/prepare/feature>` plugin
```

Comment:
I believe we have some agreed text width for documentation, I'd recommend breaking this line to be no longer than the header above.

Reply:
seems resolved? pls re-review

Reply:
@falconizmi missing full stop, sorry. Plus the referenced page should now start with a slash, `/plugins`, that changed recently.